src/index.js

```diff
@@ -0,0 +1,30 @@
+require('./index.css')
+const inject = require('./inject')
+const changeColor = require('./change-color')
+
+// invoke inject function
+inject()
+
+/**
+ * look for changes in the js-pjax-loader-bar
+ * and invoke inject function
+ * to inject github streak container
+ */
+const observer = new MutationObserver((mutations) => {
+    mutations.forEach((mutation) => {
+        if (mutation.oldValue && mutation.oldValue.indexOf("is-loading") > -1 ) {
+            // invoke inject function
+            inject()
+        }
+    })
+})
+
+/**
+ * observe pjaxLoaderBar for changes in attributes
+ */
+const pjaxLoaderBar = document.getElementById("js-pjax-loader-bar")
+const config = { attributes: true, attributeOldValue: true }
+observer.observe(pjaxLoaderBar, config)
+
+// invoke changeColor function
+changeColor()
```

Comment:
Ahh, make sense, so we can invoke `changeColor()` whenever `inject()` is being invoked.